eppoclient/client.go

```diff
@@ -80,6 +81,76 @@ func (ec *EppoClient) GetJSONAssignment(flagKey string, subjectKey string, subje
 	return variation, err
 }
 
+type BanditResult struct {
+	Variation string
+	Action    *string
+}
+
+func (ec *EppoClient) GetBanditAction(flagKey string, subjectKey string, subjectAttributes ContextAttributes, actions map[string]ContextAttributes, defaultVariation string) BanditResult {
+	// ignoring the error here as we can always proceed with default variation
+	variation, _ := ec.GetStringAssignment(flagKey, subjectKey, subjectAttributes.toGenericAttributes(), defaultVariation)
+
+	bandit, err := ec.configurationStore.getBanditConfiguration(variation)
+	if err != nil {
+		// no bandit configuration
+		return BanditResult{
+			Variation: variation,
+			Action:    nil,
+		}
+	}
+
+	evaluation := bandit.ModelData.evaluate(banditEvaluationContext{
+		flagKey:           flagKey,
+		subjectKey:        subjectKey,
+		subjectAttributes: subjectAttributes,
+		actions:           actions,
+	})
+	if evaluation.result == nil {
+		return BanditResult{
+			Variation: variation,
+			Action:    nil,
+		}
+	}
+
+	if logger, ok := ec.logger.(BanditActionLogger); ok {
+		event := BanditEvent{
+			FlagKey:                      flagKey,
+			BanditKey:                    bandit.BanditKey,
+			Subject:                      subjectKey,
+			Action:                       evaluation.result.actionKey,
+			ActionProbability:            evaluation.result.actionWeight,
+			OptimalityGap:                evaluation.result.optimalityGap,
+			ModelVersion:                 bandit.ModelVersion,
+			Timestamp:                    time.Now().UTC().Format(time.RFC3339),
```

Comment:
Just want to call out we're getting slightly consistent with our timestamp logging precission. 
* This will log to the second
* [Python SDK]("timestamp": datetime.datetime.utcnow().isoformat(),) logs the microsecond 
* [Java SDK](https://github.com/Eppo-exp/sdk-common-jdk/blob/main/src/main/java/cloud/eppo/rac/dto/BanditLogData.java#L31) keeps things as a `Date` object, which is precise to the millisecond.

I actually like the Java approach the most: keeping the date in a native datetime format and letting their logger implementation handle it. So in this case we'd just pass `time.Now().UTC()` and omit the formatting part.

Reply:
I would like to keep the `Timestamp` the same type between `AssignmentEvent` and `BanditEvent`. I'll leave a todo that we should update the type in the next major release.

Reply:
Ah yes agree that consistency would be best